posts/2023/05-06-bypassing_tasks_for_test_cases.md

```diff
@@ -0,0 +1,201 @@
+==title==
+Bypassing tasks for test cases
```

Comment:
Mocking external API calls within other processes?

When you say Task it might make someone think specifically of the Task behaviour